website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -46,7 +46,7 @@ bamboo deployment plan.
 
 #### Creating a Bamboo deployment plan
 
-1. The deployment plan is already created in Bamboo using [Bamboo Specs](https://github.com/nasa/cumulus-orca/tree/develop/bamboo-specs) 
+1. The deployment plan is already created in Bamboo using [Bamboo Specs](https://github.com/nasa/cumulus-orca/tree/develop/bamboo-specs). If you have updated the `bamboo.yaml` config file, you will need to import the updated spec file from Bamboo specs UI. Under `Specs` section, click on the `Set up Specs Repository`. On the `Project Type`, select `Build Project` and then `ORCA`. On the Specs repository, select the repository host as `orca-develop`. Note that choosing the wrong repository branch will cause issues in deployment. `ORCA repo` repository host is for `master` branch and `orca test branch` host is for `feature/ORCA-test-bamboo` branch used for testing and prototyping.

```

Comment:
I know this will hopefully go away in the future, but I would like mention of who to contact if you don't see the correct repository.

Reply:
added contact info